net/ip.c

```diff
@@ -101,15 +56,15 @@ static int m0_net_hostname_to_ip(char *hostname, char *ip,
  */
 static int m0_net_ip_inet_parse(const char *name, struct m0_net_ip_addr *addr)
 {
+	uint32_t     portnum;
```

Comment:
`uint32_t portnum` can be taken as `uint16_t portnum` as 65536 is max value hold by this variable.

Reply:
Done.